src/app/flows/fines/services/opal-fines-service/constants/opal-fines-draft-account-statues.constant.ts

```diff
@@ -4,7 +4,7 @@ export const OPAL_FINES_DRAFT_ACCOUNT_STATUSES: IOpalFinesDraftAccountStatuses =
   submitted: 'Submitted',
   rejected: 'Rejected',
   deleted: 'Deleted',
-  approved: 'Approved',
+  approved: 'PUBLISHING_PENDING',
```

Comment:
Should this not be Published?

```suggestion
  approved: 'Published',
```

Accounts will be approved by a checker and go 'Publishing Pending' and then a process in the BE changes them from 'Publishing Pending' to 'Publishing Failed' (Failed Tab) or 'Published' (Approved Tab). Might be worth checking with a BA